so5.txt

```diff
@@ -0,0 +1,75 @@
+deck_id: 1585586387
+deck_name: Systemy Operacyjne 
+file_id: wyklad-5
+
+1
+Czym jest proces?
+Wykonaniem progamu.
+
+2
+Co zawiera przestrzeń adresowa procesu?
+-kod programu <br>-dane <br>-stos procesu
+
+3
+Na czym polega translacja adresów?
+Przestrzeń adresowa jest <b>sprzętowo</b> odwzorowywana na pamięć fizyczną.
+
+4
+Jakie elementy określają przebieg wykonania procesu?
+-licznik rozkazów <br>-wskaźnik stosu <br>-rejestry
+
+5
+Gdzie system przechowuje informacje o procesach?
+W tablicy procesów.
+
+6
+Jak nazywa się element tablicy procesów?
+Blok kontrolny procesu (PCB).
+
+7
+Na czym polega podział czasu w kontekście procesów?
+Każdy proces otrzymuje kwant czasu, po jego wykorzystaniu procesor odkłada wykonanie procesu i rozpoczyna wykonywanie innego.
+
+8
+Czym jest przełączenie kontekstu?
+Odkładanie wykonania jednego procesu i rozpoczęcie wykonania innego przez procesor.
+
+9
+Jak implementowane są przerwania?
```

Comment:
to pytanie może sugerować że chodzi o to jak fizycznie dokonuje się przerwanie, znaczy że procesor odczytuje informację z pinów